src/node_kernels/SDRSSTTAMSNodeKernel.C

```diff
@@ -0,0 +1,107 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "node_kernels/SDRSSTTAMSNodeKernel.h"
+#include "Realm.h"
+#include "SolutionOptions.h"
+
+#include "stk_mesh/base/MetaData.hpp"
+#include "utils/StkHelpers.h"
+#include <SimdInterface.h>
+
+namespace sierra {
+namespace nalu {
+
+SDRSSTTAMSNodeKernel::SDRSSTTAMSNodeKernel(const stk::mesh::MetaData& meta, const std::string coordsName)
+  : NGPNodeKernel<SDRSSTTAMSNodeKernel>(),
+    dualNodalVolumeID_(get_field_ordinal(meta, "dual_nodal_volume")),
+    coordinatesID_(get_field_ordinal(meta, coordsName)),
+    tviscID_(get_field_ordinal(meta, "turbulent_viscosity")),
+    tkeNp1ID_(get_field_ordinal(meta, "turbulent_ke", stk::mesh::StateNP1)),
+    sdrNp1ID_(get_field_ordinal(
+      meta, "specific_dissipation_rate", stk::mesh::StateNP1)),
+    alphaID_(get_field_ordinal(meta, "k_ratio")),
+    fOneBlendID_(get_field_ordinal(meta, "sst_f_one_blending")),
+    dkdxID_(get_field_ordinal(meta, "dkdx")),
+    dwdxID_(get_field_ordinal(meta, "dwdx")),
+    prodID_(get_field_ordinal(meta, "average_production")),
```

Comment:
Just to confirm... the only difference here is the use of `average_density` and `average_production`?

Reply:
yup